mender-auth/ipc/server.hpp

```diff
@@ -29,21 +29,26 @@
 #include <api/api.hpp>
 #include <api/auth.hpp>
 
+#include <mender-auth/http_forwarder.hpp>
+
 namespace mender {
 namespace auth {
 namespace ipc {
 
 
 using namespace std;
 
-namespace auth_client = mender::api::auth;
 namespace conf = mender::common::conf;
 namespace dbus = mender::common::dbus;
 namespace error = mender::common::error;
 namespace events = mender::common::events;
 namespace log = mender::common::log;
 namespace path = mender::common::path;
 
+namespace auth_client = mender::api::auth;
+
+namespace http_forwarder = mender::auth::http_forwarder;
+
 class Caching {
 public:
```

Comment:
I guess now it's all too obvious this class should no longer be called `Caching`. We refer to it as `Server` anyway (see `using Server...` below) and it's much more than just a caching layer for auth data.

Reply:
True, I went for `AuthenticatingForwarder`.